packages/bpk-component-infinite-scroll/src/BpkInfiniteScroll.scss

```diff
@@ -0,0 +1,27 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-infiniteList {
+  text-align: center;
+}
+
+.bpk-sentinel {
```

Comment:
Whats a `sentinel`? :P 

Also are these styles necessary? can they safely live in userland?

Reply:
The sentinel is the element we observe in order to trigget the intersect event and load more data. I guess that this `.bpk-sentinel` is not necessary and could be defined by the user. And I guess the same for the one above. 
How would you handle this, passing a `className` property, or even letting the user define the loading component?

Reply:
Yeah allowing users to define a custom `className` is the best way to go in this case 👍